.github/CODEOWNERS

```diff
@@ -27,9 +27,11 @@
 
 /intel_extension_for_transformers/llm/runtime/deprecated/Cmake* yu.luo@intel.com
 /intel_extension_for_transformers/llm/runtime/deprecated/cmake/ yu.luo@intel.com
-/intel_extension_for_transformers/llm/libarary/jblas yu.luo@intel.com
+/intel_extension_for_transformers/llm/library/jblas yu.luo@intel.com
 /intel_extension_for_transformers/llm/runtime/graph hengyu.meng@intel.com
-/intel_extension_for_transformers/llm/libarary hengyu.meng@intel.com
+/intel-extension-for-transformers/tree/main/intel_extension_for_transformers/llm/runtime/graph/scripts zhenwei.liu@intel.com
```

Comment:
```suggestion
/intel_extension_for_transformers/llm/runtime/graph/scripts zhenwei.liu@intel.com
```
Seems that you were copying from the GitHub url :)

Reply:
can you commit it in #443 ?

Reply:
Sure~